pywt/data/_readers.py

```diff
@@ -137,3 +137,43 @@ def ecg():
     fname = os.path.join(os.path.dirname(__file__), 'ecg.npy')
     ecg = np.load(fname)
     return ecg
+
+
+def nino():
+    """
+    NINO3 sea surface temperature 1871-1996 (in C; from the UKMO GISST2.3):
```

Comment:
If we cannot resolve the license, we can always replace the data in the demo with a different, public-domain timeseries.
There are various physiological recordings available here: